src/Model/LivreManager.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sylvain
+ * Date: 07/03/18
+ * Time: 18:20
+ * PHP version 7
+ */
+
+namespace App\Model;
+
+use PDO;
+
+
+class LivreManager extends AbstractManager
+{
+    /**
+     *
+     */
+    const TABLE = 'livre';
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+
+
+    /**
+     * @param array $livre
+     * @return int
+     */
+    public function insert(array $livre): int
+    {
+        // prepared request
+        $statement = $this->pdo->prepare("INSERT INTO livre (titre, auteur, parution,
+        lecture, lu, isbn, localisation, genre, description) VALUES (:titre, :auteur , :parution,
+        :lecture, :lu, :isbn, :localisation, :genre, :description)");
+        $statement->bindValue('titre', $livre['titre'], PDO::PARAM_STR);
+        $statement->bindValue('auteur', $livre['auteur'], PDO::PARAM_STR);
+        $statement->bindValue('parution', $livre['parution'], PDO::PARAM_INT);
+        $statement->bindValue('lecture', $livre['lecture'], PDO::PARAM_INT);
+        $statement->bindValue('lu', $livre['lu'], PDO::PARAM_BOOL);
+        $statement->bindValue('isbn', $livre['isbn'], PDO::PARAM_STR);
+        $statement->bindValue('localisation', $livre['localisation'], PDO::PARAM_STR);
+        $statement->bindValue('genre', $livre['genre'], PDO::PARAM_STR);
+        $statement->bindValue('description', $livre['description'], PDO::PARAM_STR);
```

Comment:
This method calls could fail and return false. You should check it to avoid having PHP errors : method calls on false.